arkane/commonTest.py

```diff
@@ -334,25 +335,67 @@ def test_dump_yaml(self):
             job.execute(outputFile=self.dump_output_file)
         self.assertTrue(os.path.isfile(self.dump_yaml_file))
 
-    def test_load_yaml(self):
+    def test_create_and_load_yaml(self):
         """
         Test properly loading the ArkaneSpecies object and respective sub-objects
         """
-        jobList = self.arkane.loadInputFile(self.load_input_path)
+        # Create YAML file by running Arkane
+        jobList = self.arkane.loadInputFile(self.dump_input_path)
         for job in jobList:
-            job.execute(outputFile=self.load_output_file)
-        arkane_spc = jobList[0].arkane_species
+            job.execute(outputFile=self.dump_output_file)
+
+        # Load in newly created YAML file
+        arkane_spc_old = jobList[0].arkane_species
+        arkane_spc = ArkaneSpecies.__new__(ArkaneSpecies)
```

Comment:
What's the difference between this and `arkane_spc = ArkaneSpecies()`?

Reply:
I believe that `arkane_spc = ArkaneSpecies()` raises an error, because ArkaneSpecies objects need to be instantiated either with an rmgpy species object or a conformer object. We need an object in order to call the `load_yaml` method, but we don't want to try and instantiate the object until we call `load_yaml`

Reply:
Confirmed by the following:

```
>>> from arkane.common import ArkaneSpecies
>>> arkane_spc = ArkaneSpecies()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "RMG-Py/arkane/common.py", line 95, in __init__
    raise ValueError('No species (or TS) or conformer was passed to the ArkaneSpecies object')
ValueError: No species (or TS) or conformer was passed to the ArkaneSpecies object
```